### PR TITLE
Feature/GitHub template

### PR DESCRIPTION
**Pull request by markozxuu**
Added templates ❤️ 🎉 
new-issue-page-with-multiple-templates

### CODE REVIEW

**Comment by albinekb** [comment]
This is beautiful! Should we add in the part that they need to add `.hyper.js`?

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: 🦖 Bug Report
+description: Did something not work as expected ?
```
Comment (Stanzilla) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: remove the space before the ?

File: .github/ISSUE_TEMPLATE/Question.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: ❓Question
```
Comment (Stanzilla) [inline .github/ISSUE_TEMPLATE/Question.md]: add a space after the ?

File: .github/ISSUE_TEMPLATE/Question.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+description: Do you have any questions about Hyper?
+---
+
+# ❓Question
```
Comment (Stanzilla) [inline .github/ISSUE_TEMPLATE/Question.md]: same as above

**Comment by Stanzilla** [comment]
I like it 👍

File: .github/ISSUE_TEMPLATE/Question.md (hunk 3 above)
Comment (MAPESO) [reply .github/ISSUE_TEMPLATE/Question.md]: Done :)

File: .github/ISSUE_TEMPLATE/Question.md (hunk 2 above)
Comment (MAPESO) [reply .github/ISSUE_TEMPLATE/Question.md]: Done :)

File: .github/ISSUE_TEMPLATE/Bug_Report.md (hunk 1 above)
Comment (MAPESO) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: Done : )

**Comment by MAPESO** [comment]
@albinekb it would be a good idea. I can implement it when this PR is approved 😸 ❤️

**Comment by Stanzilla** [comment]
@MAPESO do you want to pick this up again?

**Comment by MAPESO** [comment]
@Stanzilla Of course ! : )

**Pull request closed without merging**